tests/func/test_repro.py

```diff
@@ -1372,210 +1373,190 @@ def test(self):
 
 
 @pytest.fixture
-def repro_dir(dvc_repo, repo_dir):
-    repo_dir.dname = "dir"
-    os.mkdir(repo_dir.dname)
-    repo_dir.emptydname = "emptydir"
-    os.mkdir(repo_dir.emptydname)
-    subdname = os.path.join(repo_dir.dname, "subdir")
-    os.mkdir(subdname)
-
-    repo_dir.source = "source"
-    repo_dir.source_stage = repo_dir.source + ".dvc"
-    stage = dvc_repo.run(
-        fname=repo_dir.source_stage,
-        outs=[repo_dir.source],
-        deps=[repo_dir.FOO],
-    )
-    assert stage is not None
-    assert filecmp.cmp(repo_dir.source, repo_dir.FOO, shallow=False)
-
-    repo_dir.unrelated1 = "unrelated1"
-    repo_dir.unrelated1_stage = repo_dir.unrelated1 + ".dvc"
-    stage = dvc_repo.run(
-        fname=repo_dir.unrelated1_stage,
-        outs=[repo_dir.unrelated1],
-        deps=[repo_dir.source],
-    )
-    assert stage is not None
-
-    repo_dir.unrelated2 = "unrelated2"
-    repo_dir.unrelated2_stage = repo_dir.unrelated2 + ".dvc"
-    stage = dvc_repo.run(
-        fname=repo_dir.unrelated2_stage,
-        outs=[repo_dir.unrelated2],
-        deps=[repo_dir.DATA],
+def repro_dir(tmp_dir, dvc, run_copy):
+    # Creates repo with following structure:
+    #    data_dir/dir_file              origin_data
+    #         |       |                   |
+    #         |       |              origin_copy.dvc
+    # unrelated2.dvc  |               |       |
+    #                 |               |    unrelated1.dvc
+    #    dir/subdir/dir_file_copy.dvc |
+    #                  |              |
+    #                  |        dir/origin_copy_2.dvc
+    #                  |            |
+    #                   \          /
+    #                    \        /
+    #                   dir/Dvcfile
```

Comment:
You could've used `tree` CLI util, it would be much easier to understand than graph 🙂

Reply:
I intended to show dependency between particular stages, `tree` would show placement in the directory. I thought that this one brings more information. Maybe I should include both?

Reply:
Oh, right, sorry. Ok, why not `dvc pipeline show --ascii` then? Or is it too big?

Reply:
^ Just wondering, I'm okay with this illustration.

Reply:
@efiop, you are right, it was too big, had to to it my own way, so that we will still be able to pass all checks :)